src/frequenz/sdk/actor/_power_managing/_matryoshka.py

```diff
@@ -49,6 +52,21 @@ def get_target_power(
                 already part of another bucket.
         """
         if battery_ids not in self._battery_buckets:
+            # if there are no previous proposals and there are no system bounds, then
+            # don't calculate a target power and just return `None`.
+            if (
+                system_bounds.inclusion_bounds is None
+                and system_bounds.exclusion_bounds is None
+            ):
+                if proposal is not None:
+                    _logger.warning(
+                        "PowerManagingActor: No system bounds available for battery "
+                        "IDs %s, but a proposal was given.  The proposal will be "
+                        "ignored.",
+                        battery_ids,
+                    )
+                return None
```

Comment:
Could it happen that someone sets power before the system bounds are initialized? Is there a way to make sure they are initialized before using?

Reply:
yes, I think the trick is for actors to get the bounds first, ensure their requests comply with the bounds, then make the proposal.

but even if people don't do that,  only the first request will get ignored (and logged) because there are no bounds.  because as soon as a proposal comes in, we start tracking bounds for that set of batteries.